package.json

```diff
@@ -2,7 +2,7 @@
   "private": true,
   "scripts": {
     "lint": "eslint --ext .js,.mjs,.ts,.vue,.astro .",
-    "linkinator": "linkinator dist/ --recurse --verbosity error --skip 'twitter.com|www.farmlands.co.nz|www.instagram.com|yourapp.example.com|centrapay-docs.dev.s3-website-ap-southeast-1.amazonaws.com/api'",
+    "linkinator": "linkinator dist/ --recurse --verbosity error --skip 'twitter.com|www.farmlands.co.nz|www.instagram.com|yourapp.example.com|centrapay-docs.dev.s3-website-ap-southeast-1.amazonaws.com/api|https://www.verifone.com/en/nz|https://www.gs1.org/standards/gpc'",
```

Comment:
Why are we skipping link check on these two new links?

Reply:
The linkinator is failing on master branch currently (not related to your change)
[2:16](https://centrapay.slack.com/archives/D048MV2159V/p1719368218615909)
If you go to https://www.verifone.com/en/nz there is a 404 in the console which could be the reason why linkinator is failing - when in actual fact there is nothing wrong with the website
Verifone
[Verifone - USA | Verifone](https://www.verifone.com/en/nz)
Simply shaping reliable payment ecosystems for an ever evolving world of commerce.